card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
it looks to me like they don't.  they should; that would take care of most of this.  perhaps #menu_item should add the slotter class by default.  I'm guessing the page link is the only place we don't want it.

Then you could probably get rid of #menu_path_opts altogether.